todo_app/data/trello_items.py

```diff
@@ -2,33 +2,29 @@
 from datetime import datetime
```

Comment:
You will want to change the name of files and functions. This is no longer a 'trello' related functionality